zad6/db/migrate/20181210173913_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[5.2]
+  def change
+	create_table :products do |t|
+		t.string :name, null: false
+		t.decimal :price, precision: 10, scale: 2, null: false
+		t.index :name, unique: true
```

Comment:
Jeśli tworzymy unikalny index na kolumnie `name` to dobrze by było napisać też walidację w odpowiadającym modelu. To sprawia, że otrzymujemy błędy z modelu, które jesteśmy w stanie odpowiednio obsłużyć i poinformować o nich użytkownika. Obecnie, jeśli spróbujemy stworzyć produkt z `name`, który jest już w bazie dostaniemy coś takiego:
![screen shot 2018-12-11 at 15 51 15](https://user-images.githubusercontent.com/18199403/49808348-a0b78380-fd5c-11e8-989c-2650f3838e7b.png)